src/components/Calculator.js

```diff
@@ -1,44 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
 // eslint-disable import/no-extraneous-dependencies
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends React.Component {
   // eslint-disable-next-line no-useless-constructor
   constructor(props) {
     super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.handleClick.bind(this);
   }
 
+  handleClick = (e) => {
+    const buttonName = e.target.textContent;
+    const { total, next, operation } = this.state;
+    const obj = calculate(({ total, next, operation }), buttonName);
+
+    this.setState(() => ({
+      total: obj.total,
+      next: obj.next,
+      operation: obj.operation,
```

Comment:
- [ ] please handle your state using the useState hook.

Reply:
Hi Mike, this project for yesterday requirements, so it is not supposed to use useState for it. 
Notice here [yesterday project](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_events.md).
Yesterday project requirements:

Create a new directory called logic inside your project
Copy the files calculate.js and operate.js into your logic/ directory
Install [big.js](https://www.npmjs.com/package/big-js) using NPM
Analyze the files calculate.js and operate.js, be sure you understand what they do
Import the files in your Calculator.js component
Implement click event handlers using the math logic from calculate.js and operate.js in your React component
Test your app in the browser. The Calculator should be able to perform math operations.
Note: Practice what you have learned until now: use useState and props

Please review my project again so as to go to next step. Thank you @mikethreels